components/layout/flow/inline/mod.rs

```diff
@@ -2313,6 +2318,14 @@ struct ContentSizesComputation<'layout_data> {
     current_line: ContentSizes,
     /// Size for whitespace pending to be added to this line.
     pending_whitespace: ContentSizes,
+    /// The size of the not yet cleared floats on the inline-start side of the containing block.
+    uncleared_start_floats: ContentSizes,
+    /// The size of the already cleared floats on the inline-start side of the containing block.
+    cleared_start_floats: ContentSizes,
+    /// The size of the not yet cleared floats on the inline-end side of the containing block.
+    uncleared_end_floats: ContentSizes,
+    /// The size of the already cleared floats on the inline-end side of the containing block.
+    cleared_end_floats: ContentSizes,
```

Comment:
A small point but these could be `LogicalSides1D`:


```suggestion
    /// The size of the not yet cleared floats in the inline axis of the containing block.
    uncleared_floats: LogicalSides1D<ContentSizes>,
    /// The size of the already cleared floatsin the inline axis of the containing block.
    cleared_floats: LogicalSides1D<ContentSizes>,
```